src/OrchardCore.Modules/Orchard.Autoroute/Handlers/AutoroutePartHandler.cs

```diff
@@ -100,7 +100,11 @@ public override void Updated(UpdateContentContext context, AutoroutePart part)
 
             if (!String.IsNullOrEmpty(pattern))
             {
-                part.Path = _tokenizer.Tokenize(pattern, new Dictionary<string, object> { ["Content"] = part.ContentItem });
+                var templateContext = new TemplateContext();
+                templateContext.SetValue("ContentItem", part.ContentItem);
+
+                part.Path = _liquidTemplateManager.RenderAsync(pattern, templateContext).GetAwaiter().GetResult();
```

Comment:
I think we already talked about it, yes. I will file an issue, can be part of another PR.